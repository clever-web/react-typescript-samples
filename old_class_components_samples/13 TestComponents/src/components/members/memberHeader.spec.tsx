import * as React  from 'react';
import {MemberHeader} from './memberHeader';
import {shallow} from 'enzyme';

describe('./components/members/memberHeader',()=>{
    it('should render memberHeader as expected',()=>{
        //arrange
        //act
        const component =  shallow(
            <MemberHeader />
        );
        //Assert
        expect (component).toMatchSnapshot();

    });
});