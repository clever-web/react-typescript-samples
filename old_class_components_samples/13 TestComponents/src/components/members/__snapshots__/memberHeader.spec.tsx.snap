// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./components/members/memberHeader should render memberHeader as expected 1`] = `
<tr>
  <th>
    Avatar
  </th>
  <th>
    Id
  </th>
  <th>
    Name
  </th>
</tr>
`;
